src/Components/source.js

```diff
@@ -22,7 +22,7 @@ class Source extends Component {
 
     let cardDisplay = this.props.data.map((post) => {
       return (
-        <Grid.Column width={3}>
+        <Grid.Column width={3} style={{ marginLeft: '8px' }}>
```

Comment:
For component segmentation sake, could you please move this "style definition" into the Article component instead?

Reply:
I did